test/test_board.py

```diff
@@ -1,43 +1,286 @@
 import unittest
+
+from parameterized import parameterized
+
 from constans.constans import (
+    INITIAL_ARROWS,
     INITIAL_POSITION_PLAYER_1,
     INITIAL_POSITION_PLAYER_2,
     PLAYER_1,
     PLAYER_2,
 )
+from constans.constants_game import LARGE
+from constans.constants_scores import (
+    ARROW_MISS,
+    CORRECT_MOVE,
+    KILL
+)
+from constans.constants_utils import NORTH, SOUTH, EAST, WEST
+from exceptions.personal_exceptions import (
+        friendlyFireException,
+        noArrowsAvailableException,
+        shootOutOfBoundsException
+)
 from game.board import Board
+from game.character import Character
+from game.diamond import Diamond
+from game.gold import Gold
 from game.player import Player
 
-from constans.constants_game import LARGE
-
 
-class TestGame(unittest.TestCase):
+class TestBoard(unittest.TestCase):
 
     def test_board_size(self):
```

Comment:
What does test_board_size mean?
It could be instead something like:
`test_board_is_created_with_right_dimensions`